lib/transports/rest2.js

```diff
@@ -250,6 +254,36 @@ class RESTv2 {
     return new ModelClass(data)
   }
 
+  /**
+   * @param {Arr} data order matters
+   * @return {Arr} merged arr of coins and names with no pairs rpeated
+   * @private
+   */
+  _mergeAllArrOfCoins (data) {
+    const addKeyToObj = (obj, curr) => {
+      if (!Array.isArray(curr)) obj[curr] = curr
+      else if (curr.length === 1) obj[curr[0]] = curr[0]
+      else obj[curr[0]] = curr[1]
```

Comment:
Please add brackets around all clauses, and put them on newlines separate from the conditions